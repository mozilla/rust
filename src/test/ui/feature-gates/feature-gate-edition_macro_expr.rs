// feature gate test for edition_macro_expr

macro_rules! mac {
    ($x:expr2015) => {}; //~ERROR `expr2015` and `expr202x` are unstable
    ($x:expr202x) => {}; //~ERROR `expr2015` and `expr202x` are unstable
}

fn main() {}
