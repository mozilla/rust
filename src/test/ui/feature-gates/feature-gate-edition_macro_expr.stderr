error[E0658]: `expr2015` and `expr202x` are unstable.
  --> $DIR/feature-gate-edition_macro_expr.rs:4:9
   |
LL |     ($x:expr2015) => {};
   |         ^^^^^^^^
   |
   = help: add `#![feature(edition_macro_expr)]` to the crate attributes to enable

error[E0658]: `expr2015` and `expr202x` are unstable.
  --> $DIR/feature-gate-edition_macro_expr.rs:5:9
   |
LL |     ($x:expr202x) => {};
   |         ^^^^^^^^
   |
   = help: add `#![feature(edition_macro_expr)]` to the crate attributes to enable

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
